go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/unidoc/freetype
+
+go 1.17
+
```

Comment:
@gunnsth seems need to put the replace 
`github.com/golang/freetype => ../freetype`